### ISSUE TITLE
No distance and time in activities

### ISSUE THREAD

**Issue by ferferga** [praveendath92/fitbit-googlefit#40]
First of all, thank you for the script, is really useful

But I found some issues: Steps and calories are synced correctly to Google Fit, but although the script reports me that also the distance has been synced, that's not true. I can't see any distance or time of activity. I use Fitbit with MobileTrack for counting the daily steps without starting any monitoring in the fitbit app, just for counting the steps, distance and estimated calories.

There is an screenshot showing how the calories are being synced
![captura](https://user-images.githubusercontent.com/10274099/31583090-7433f97a-b194-11e7-9d7a-30229d2b6a8f.PNG)

Is strange how Google Fit works (I've already seen reports of this) but for the same day, in the calories view there are calories and in the step view there isn't.

![captura](https://user-images.githubusercontent.com/10274099/31583142-f727b6aa-b194-11e7-81b6-190df054924c.PNG)

This isn't happening for the distance. If I switch to distance view, there is no distance at all. I thought that this script would sync the GPS positions in order to see the places I've been to also in Google Fit.

Thank you very much for this script!

**Comment by praveendath92** [comment]
Hi, from what I noticed, Google Fit is only showing distance covered during activities in the distance view. Also, the script doesn't use GPS positions for distance sync. It takes distance values reported by Fitbit for each minute and syncs them.

**Comment by ferferga** [comment]
Yeah, that's what I wanted to explain but I think I couldn't do it well. As you can see in the screenshots, only the calories can be seen in the calories view, the steps only can be seen in the step view... But that's not the case for the distance: That's what I get:

![captura](https://user-images.githubusercontent.com/10274099/31584126-94e77778-b1a8-11e7-928f-c6ba52f8e816.PNG)

I suppose the GPS positions aren't synced because Fitbit doesn't record them, am I right? If it does, it would be great if this script could sync that too, as the precision will be even better.

Thank you very much again!
